packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -581,8 +504,6 @@ contract SphinxUtils is SphinxConstants, StdUtils {
         );
 
         address[] memory duplicateOwners = getDuplicatedElements(_config.owners);
-        Network[] memory duplicateMainnets = getDuplicatedElements(_config.mainnets);
-        Network[] memory duplicateTestnets = getDuplicatedElements(_config.testnets);
```

Comment:
Moved the network validation in this file to the `validateProposalNetworks` TypeScript function to reuse the validation logic that I wrote for the arbitrary network names passed via the CLI